### ISSUE TITLE
Reading Parquet from s3

### ISSUE THREAD

**Issue by gregleleu** [neo4j-contrib/neo4j-apoc-procedures#3823]
## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use APOC, please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/neo4j-graph-platform/procedures-apoc) instead of creating an issue here.

## Expected Behavior (Mandatory)
Reading parquet from s3 bucket

## Actual Behavior (Mandatory)
Exception:
org.apache.hadoop.fs.UnsupportedFileSystemException: No FileSystem for scheme "s3"

## How to Reproduce the Problem
Try apoc.load.parquet on s3 address

```
CALL apoc.load.parquet("s3://...") YIELD value
RETURN value 

```

## Specifications (Mandatory)
Reading from s3 works from other procedures, esp. apoc.load.csv

Not clear which jars should be present from the documentation, I have:
- from the release page: 
  - apoc-aws-dependencies-5.13.0.jar 
  - apoc-hadoop-dependencies-5.13.0-all.jar
- from the documentation: 
  - aws-java-sdk-core-1.12.136.jar
  - aws-java-sdk-s3-1.12.136.jar
  - httpclient-4.5.13.jar
  - httpcore-4.4.15.jar
  - joda-time-2.10.13.jar

### Versions

  - OS: Docker image
  - Neo4j: 5.13
  - Neo4j-Apoc: Extended 5.13

**Comment by conker84** [comment]
@gregleleu we're investigating the problem, thank you so much for reporting this. We'll be back ASAP

**Comment by vga91** [comment]
We have solved the issue [here](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3846).

The fix is available from version 5.14.0
